backend/inspirehep/search/api.py

```diff
@@ -208,38 +198,35 @@ def query_from_iq(self, query_string):
         :returns: Elasticsearch DSL search class
         """
         if not is_superuser_or_cataloger_logged_in():
-            user_query = Q(IQ(query_string, self) & Q("term", status="open"))
+            if not query_string:
+                user_query = Q("term", status="open")
+            else:
+                user_query = Q(
+                    Q("query_string", query=query_string) & Q("term", status="open")
+                )
             return self.query(user_query)
-        return self.query(IQ(query_string, self))
+        return super().query_from_iq(query_string)
```

Comment:
```suggestion
        return super().query_from_iq(user_query)
```
how are your new tests passing despite this?

Reply:
No!